lib/pos_blockchain.go

```diff
@@ -181,10 +181,14 @@ func (bc *Blockchain) validateBlockGeneral(desoBlock *MsgDeSoBlock) error {
 
 // validateBlockHeight validates the block height for a given block. It checks
 // that this block height is exactly one greater than the current block height.
-// TODO: Are we sure that's the correct validation here?
 func (bc *Blockchain) validateBlockHeight(desoBlock *MsgDeSoBlock) error {
-	// TODO: Implement me
-	return errors.New("IMPLEMENT ME")
+	if desoBlock.Header.Height != bc.GetBestChainTip().Header.Height+1 {
```

Comment:
This should make sure that the block's height is +1 of its parent's block height. This also addresses the fork question below.